src/hooks/usePlanByEpg.ts

```diff
@@ -13,13 +13,21 @@ const isBaseTimeFormat = is12HourClock();
 const usePlanByEpg = (channels: EpgChannel[], sidebarWidth: number, itemHeight: number, highlightColor?: string | null, backgroundColor?: string | null) => {
   const [epgChannels, epgPrograms] = useMemo(() => {
     return [
-      channels.map((channel) => ({ uuid: channel.id, logo: channel.image })),
+      channels.map((channel) => ({
+        uuid: channel.id,
+        logo: channel.channelLogoImage?.image || '',
+        channelLogoImage: channel.channelLogoImage,
+        backgroundImage: channel.backgroundImage,
+      })),
       channels.flatMap((channel) =>
         channel.programs.map((program) => ({
           channelUuid: channel.id,
           id: program.id,
           title: program.title,
-          image: program.image || '',
+          image: program.shelfImage?.image || '',
```

Comment:
Does it make sense to use the fallback image here?

Reply:
We don't use the `image` property, but it is required by planby.